crates/namada/src/vm/host_env.rs

```diff
@@ -2537,10 +2557,14 @@ where
     H: StorageHasher,
     CA: WasmCacheAccess,
 {
-    fn write_bytes(
+    // TODO: is it ok to not explicitly use the `_actions` and use the default
+    // behavior????
```

Comment:
I think we should use `write_log.write_with_opts/delete_with_opts` otherwise the opts won't work in a IBC tx